components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -67,7 +67,7 @@ func NewProvision(cfg Config, operationsStorage storage.Operations, instanceStor
 func (b *ProvisionEndpoint) Provision(ctx context.Context, instanceID string, details domain.ProvisionDetails, asyncAllowed bool) (domain.ProvisionedServiceSpec, error) {
 	operationID := uuid.New().String()
 	logger := b.log.WithField("instanceID", instanceID).WithField("operationID", operationID)
-	logger.Infof("Provision called: planID=%s", details.PlanID)
+	logger.WithField("planID", details.PlanID).Info("Provision called")
```

Comment:
if you are not using planID as a field in the next logging, leave it as it was before